### ISSUE TITLE
When I was using this pretraining.py to pretrain bert, I found a strange loss

### ISSUE THREAD

**Issue by 652994331** [NVIDIA/DeepLearningExamples#401]
At the  first a few steps, the loss was 7.*  . later, it fall to 2.3(after around 30000 steps)
However, after 200000 steps, the loss increased to 7.x . it's not a slow increase but a sharp one, as you can see from the log below:
![image](https://user-images.githubusercontent.com/51428350/74560532-9de58f00-4f1b-11ea-86dc-fe265a70d658.png)
it's so strange, could you please explain the reason? thanks so much

**Comment by swethmandava** [comment]
You can try to smooth the loss over a few steps like [here](https://github.com/NVIDIA/DeepLearningExamples/blob/0845aaa90187e0b79d244648deb2c9656bd6fb66/TensorFlow/LanguageModeling/BERT/run_pretraining.py#L179) to get a clearer picture. It could just be a bad batch

**Comment by 652994331** [comment]
@swethmandava thanks, got u
